cluster.tf

```diff
@@ -499,6 +499,11 @@ resource "google_container_cluster" "primary" {
       }
     }
   }
+
+  node_pool_defaults {
+    node_config_defaults {
+    }
```

Comment:
Note: this is obviously useless as it is. but idea is to pave the way for #2082 once some of the other upstream issues are resolved.

Reply:
Yup, I'm also interested in the "test coverage".  The alternative would be to make this a dynamic block.